pages/politicians/[slug].tsx

```diff
@@ -81,6 +81,9 @@ const PoliticianPage: NextPage<{ mobileNavTitle?: string }> = ({
   const ratings = politician?.ratings.edges as Array<RatingResultEdge>;
   const politicalExperience =
     politician?.votesmartCandidateBio?.candidate?.congMembership?.experience;
+  
+  // const raceWins = politician?.race_wins;
+  // const raceLosses = politician?.race_losses;
```

Comment:
lets delete until we have this data available.  also, any data on the front end will be camelCase, not snake_case